homeassistant/components/tuya/switch.py

```diff
@@ -765,7 +765,10 @@ def __init__(
     @property
     def is_on(self) -> bool:
         """Return true if switch is on."""
-        return self.device.status.get(self.entity_description.key, False)
+        state = self.device.status.get(self.entity_description.key, False)
+        if isinstance(state, str):
+            state = state.lower() in ["true", "on", "1"]
+        return state
```

Comment:
Same comment as the first one. This seems to be off specification.

Reply:
Same comment as above.

Reply:
Although off specification, but can we temporarily do this. As mentioned in the previous comment, Tuya is unresponsive for aome time.